get_forecast.py

```diff
@@ -324,60 +324,57 @@ def calc_forecast(boto3_session):
     return output
 
 
-def format_rows(output,account_width):
-    #print the heading
-    mtd_width=8
-    forecast_width=8
-    change_width=6
+def format_rows(output, account_width):
+    # print the heading
+    forecast_width = 8
+    change_width = 8
 
-    output_rows=[]
+    total_forecast = 0
+    total_change = 0
+    
+    output_rows = []
 
-    row = {
-        "Account": 'Account'.ljust(account_width),
-        "MTD": 'MTD'.rjust(mtd_width),
```

Comment:
incorrect to remoe MTD here, you should only modify the display the code is designed for it to be configurabel.  before making a change to remove the feature you should discuss.. you simply supresss in line 361